### PR TITLE
sys-apps/util-linux: update to 2.37.4

### PR DESCRIPTION
**Pull request by dongsupark**
Sync with Gentoo to update `sys-apps/util-linux` to 2.37.4, mainly to address the following security issues:

* [CVE-2021-3995](https://nvd.nist.gov/vuln/detail/CVE-2021-3995)
* [CVE-2021-3996](https://nvd.nist.gov/vuln/detail/CVE-2021-3996)
* [CVE-2022-0563](https://nvd.nist.gov/vuln/detail/CVE-2022-0563)

Written by @sayanchowdhury.

This PR should be merged together with https://github.com/flatcar-linux/coreos-overlay/pull/1745.

## Testing done

CI passed: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/5192/cldsv

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

File: sys-apps/util-linux/files/su-l.pamd [hunk 1]
```diff
@@ -0,0 +1,5 @@
+auth	   include      su
+account    include      su
+password   include      su
+session	   optional pam_lastlog.so
+session	   include      su
```
Comment (pothos) [inline sys-apps/util-linux/files/su-l.pamd:5]: I assume this doesn't require util-linux's su binary

Comment (pothos) [reply sys-apps/util-linux/files/su-l.pamd:5]: I'll try to test it manually and see if something is off

Comment (pothos) [reply sys-apps/util-linux/files/su-l.pamd:5]: `su -l -c whoami - root` from user core and `su -l -c whoami - core` from user core both worked with the respective password set

**Review by pothos — APPROVED** [review]

**Comment by dongsupark** [comment]
Thanks for the review.
Will then merge the PR for the next Alpha.

**Pull request merged**